pkg/controller/allowlist/allowlist_controller.go

```diff
@@ -76,6 +97,7 @@ func (r *ReconcileAllowlist) Reconcile(ctx context.Context, request reconcile.Re
 	if request.Namespace != names.MULTUS_NAMESPACE || request.Name != names.ALLOWLIST_CONFIG_NAME {
```

Comment:
I suppose we can delete the namespace check

Reply:
Ok, will do

Reply:
Done